so_pysm_models/tests/test_co.py

```diff
@@ -0,0 +1,32 @@
+from astropy.utils.data import get_pkg_data_filename
+
+import numpy as np
+import healpy as hp
+
+import pytest
+
+from .. import COLines
+
+@pytest.mark.parametrize("include_high_galactic_latitude_clouds", [False, True])
+def test_co(include_high_galactic_latitude_clouds):
+
+    co = COLines(target_nside=16,
+        has_polarization=True,
+        line="10",
+        include_high_galactic_latitude_clouds=include_high_galactic_latitude_clouds,
+        polarization_fraction=0.001,
+        theta_high_galactic_latitude_deg=20.,
+        random_seed=1234567,
+        verbose=False,
+        run_mcmole3d=False
+    )
```

Comment:
@giuspugl the test doesn't pass. I tried also with theta=0, can you please take a look and see if you find a bug in my code?

You could also checkout this repository and run the code yourself, thanks.

Reply:
Ok found the error and fixed it, now tests pass i am gonna make a pull request to this branch with  more documentation and the details on the beams.